operator/v1/types_network.go

```diff
@@ -429,6 +430,15 @@ type HybridOverlayConfig struct {
 }
 
 type IPsecConfig struct {
```

Comment:
This needs a CEL validation now
```suggestion
// +kubebuilder:validation:XValidation:rule="self == oldSelf || self.has(mode)",message="ipsecConfig.mode is required"
type IPsecConfig struct {
```